docs/auth/github/provider.md

```diff
@@ -50,6 +52,10 @@ The GitHub provider is a structure with three configuration keys:
 - `clientSecret`: The client secret tied to the generated client ID.
 - `enterpriseInstanceUrl` (optional): The base URL for a GitHub Enterprise
   instance, e.g. `https://ghe.<company>.com`. Only needed for GitHub Enterprise.
+- `callbackUrl` (optional): The callback url that GitHub will use when
```

Comment:
Tbh I think this is such a niche use case that we should only document it here avoid skip the two examples above and in the other doc. It's unfortunate if people end up struggling with configuring the callback URL when there's no need

Reply:
Ok sounds good.

Reply:
Done!